src/browser.ts

```diff
@@ -90,11 +120,12 @@ export function parseAndValidateBrowser(str: string): BrowserConfig {
   if (headless === true && !headlessBrowsers.has(name)) {
     throw new Error(`Browser ${name} does not support headless mode.`);
   }
-  const config: BrowserConfig = {name, headless};
-  if (remoteUrl !== undefined) {
-    config.remoteUrl = remoteUrl;
+  if (remoteUrl !== undefined && !isHttpUrl(remoteUrl)) {
+    throw new Error(`Invalid browser remote URL "${remoteUrl}".`);
+  }
+  if (windowSize.width < 0 || windowSize.height < 0) {
```

Comment:
This checks < but the message says <=

The JSON schema says minimum of 0, so < would be the most consistent.

(I'm standing up for the zero-sized browser community here. It's the next step beyond no-js, no-images, no-css. just embed a WAV file in an `<audio>` tag)

Reply:
:)

(actually it says >=0 since it's phrased the other way around)